api/tests/pokemon.test.js

```diff
@@ -15,34 +15,37 @@ describe('GET /pokemon', () => {
     expect(typeof pokemon.body).toBe('object')
   })
   it('should limit unspecified requests to 20', async () => {
-    //https://pokeapi.co/api/v2/pokemon?offset=20&limit=20
     const pokemon = await Posts.getPokemans(path)
     expect(pokemon.body.results.length).toBe(20)
   })
   it('should allow requests to be > 20', async () => {
-    //https://pokeapi.co/api/v2/pokemon?offset=20&limit=20
     path = `pokemon?offset=20&limit=80`
     const pokemon = await Posts.getPokemans(path)
     expect(pokemon.body.results.length).toBe(80)
   })
   it('should use the offset to remove the first x number of results', async () => {
-    //https://pokeapi.co/api/v2/pokemon?offset=20&limit=20
     path = `pokemon?offset=40&limit=80`
     const pokemon = await Posts.getPokemans(path)
     expect(pokemon.body.results.length).toBe(80)
   })
   it('should use the offset to remove the first x number of results', async () => {
-    //https://pokeapi.co/api/v2/pokemon?offset=20&limit=20
-    path = `pokemon?offset=40&limit=40` //rewrite this to validate that 40/40 does gives us pokemon 41-80, not 0-40
+    path = `pokemon?offset=40&limit=10`
     const pokemon = await Posts.getPokemans(path)
-    expect(pokemon.body.results.length).toBe(40)
+    const pokemonIDs = pokemon.body.results.map(function(pokemon){
+      return pokemon.url.slice(-3).slice(0, -1)
+    })
+
+    const control = [
+      '41', '42', '43',
+      '44', '45', '46',
+      '47', '48', '49',
+      '50'
+    ]
+    expect(pokemon.body.results.length).toBe(10)
```

Comment:
This first assertion is redundant, lets remove it.

Reply:
thanks good catch!